src/toolkits/drawing_classifier/data_preparation.cpp

```diff
@@ -390,7 +439,7 @@ flex_image rasterize_on_mac(flex_list simplified_drawing) {
 }
 #endif // __APPLE__
 
-flex_image rasterize(flex_list simplified_drawing) {
+flex_image rasterize(flex_list &simplified_drawing) {
```

Comment:
Needs to either be const flex_list& or have documentation explaining how it's changed.  It's not clear to me why it is changed.